contracts/eosio.system/src/voting.cpp

```diff
@@ -23,14 +23,6 @@ namespace eosiosystem {
    using eosio::singleton;
    using eosio::transaction;
 
-   /**
```

Comment:
Why are you removing these? They seem like good comments to have.

Reply:
they are annotated in the hpp file; if we leave these annotations here they will double up in the final generated doc.